rrmngmnt/db.py

```diff
@@ -40,6 +40,10 @@ def psql(self, sql, *args):
             raise Exception(
                 "Failed to exec sql command: %s" % err
             )
+        # from PG10 could print output to err even when rc = 0
```

Comment:
:-O that is not very convenient in my opinion ... could you please give me example of command which reproduce that ? Thank you.

Reply:
I agree with that, but that is how PG10 works now. 
try for example: psql -c '\dt' <newly created database>